tools/bin/commands/git

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+git::description() {
+    echo "Git tools"
+}
+
+git::usage() {
+    cat <<EOT
+    has_app_module_changed <module>     prints true if the module has changed relative to master
+    touched_pl                          prints the list of /app modules uncommitted changes
```

Comment:
perhaps we can make this command look like other commands (use `-` instead of `_`), add shorthand (one letter) parameters? this could be as simple as `-c/--has-changed` and `-s/--show-changed`

Reply:
Could we also add a '-q' or '--quiet' which instead of printing 'true' or 'false' just exit with 0 or 1 ? We could put this kind of function in `util/github_funcs` and then e.g. reuse it for "build" to offer an option only to rebuild those modules that have changed (+ the one that depend on those modules).

Also, instead of just putting out a change it would might make sense also to tell *what* has changed (by whom).